app/views/idv/shared/_document_capture.html.erb

```diff
@@ -38,6 +38,8 @@
       in_person_outage_expected_update_date: IdentityConfig.store.in_person_outage_expected_update_date,
       us_states_territories: us_states_territories,
       doc_auth_selfie_capture: IdentityConfig.store.doc_auth_selfie_capture,
+      skip_doc_auth: skip_doc_auth,
+      how_to_verify_url: idv_how_to_verify_url,
```

Comment:
Is this `how_to_verify_url` parameter still needed? I didn't see it used in this PR.

Reply:
@soniaconnolly I just confirmed this is needed.  in-person-prepare-step.tsx uses in goBack. When you click the Back link on /verify/document_capture#prepare it will take you back to /verify/how_to_verify (if skipDocAuth = true).

When I deleted it, howToVerifyURL inside in-person-prepare-step.tsx is undefined and the Back link does nothing.